model/enduser/registry.yaml

```
@@ -0,0 +1,24 @@
+groups:
+  - id: registry.enduser
+    type: attribute_group
+    display_name: End User Attributes
+    brief: >
+      Describes the end user.
+    attributes:
+      - id: enduser.id
+        type: string
+        brief: "Unique identifier of an end user in the system. It maybe a username, email address, or other identifier."
+        note: |
+          > [!Warning]
+          > This field contains sensitive (PII) information.
```

i'm not sure, but try adding a newline here to fix the rendering issue:

```suggestion
        note: |

          > [!Warning]
          > This field contains sensitive (PII) information.
```